hw3/01-characters.js

```diff
@@ -1,4 +1,24 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    for (const character of data) {
+      const card = document.createElement('card');
+      const image = document.createElement('img');
+      image.src = character.imageUrl;
+      image.height = '200';
+      image.width = '200';
+      card.appendChild(image);
+      const listItem = document.createElement('h2');
+      listItem.append(character.firstName + ' ' + character.lastName);
+      card.appendChild(listItem);
+      const subtitle = document.createElement('h3');
+      subtitle.append(character.title);
+      card.appendChild(subtitle);
+      section.appendChild(card);
+    }
+  })
+  .catch(console.error);
```

Comment:
Oh I totally forgot about using the catch for my code.  One thing that might be better would be to display something that the user can actually see instead of just console logging.